test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -93,14 +80,13 @@ namespace iroha {
           const shared_model::interface::types::RoleIdType role_id = "all") {
         shared_model::interface::RolePermissionSet permissions;
         permissions.set();
-        ASSERT_TRUE(
-            val(execute(buildCommand(TestTransactionBuilder().createRole(
-                            role_id, permissions)),
-                        true)));
-        ASSERT_TRUE(
-            val(execute(buildCommand(TestTransactionBuilder().appendRole(
-                            account_id, role_id)),
-                        true)));
+
+        ASSERT_TRUE(val(execute(
```

Comment:
This construct repeats a lot, please move it to separate function,
so it is possible to write:
```suggestion
        ASSERT_TRUE(execute(mock_command_factory->contructWhatever(...)));
```